terraform/account-wide-infrastructure/prod/locals.tf

```diff
@@ -3,4 +3,6 @@ locals {
   project     = "nhsd-nrlf"
   environment = terraform.workspace
   prefix      = "${local.project}--${local.environment}"
+
+  notification_emails = nonsensitive(toset(tolist(jsondecode(data.aws_secretsmanager_secret_version.emails.secret_string))))
```

Comment:
I see this is the same as the dev account. So I understand for consistency it should be like this but it feels like this shouldn't be marked as nonsensitive?